studio/src/lib/schema-helpers.ts

```diff
@@ -546,6 +539,39 @@ export const getGraphQLTypeAtLineNumber = (
   return null;
 };
 
+const maybeParseField = (field: GraphQLField<any, any> | GraphQLInputField): ParsedGraphQLField | null => {
```

Comment:
Is it really necessary for all these types to be so wide?

Reply:
This is the tightest I could make it becasue the type found by `searchSchema` can be an input object